codecov.yaml

```diff
@@ -2,15 +2,22 @@ coverage:
   status:
     project:
       default:
-        # Commits pushed to master should not make the overall
-        # project coverage decrease by more than 1%:
-        target: auto
-        threshold: 1%
-    patch:
-      default:
-        # Be tolerant on slight code coverage diff on PRs to limit
-        # noisy red coverage status on github PRs.
-        # Note: The coverage stats are still uploaded
-        # to codecov so that PR reviewers can see uncovered lines
+        # by default, we only care about test coverage of the main
+        # rust crates
         target: auto
         threshold: 1%
+        paths:
+          - "crates"
+      bindings:
+        # Coverage of binding tests is recorded but for informational
+        # purposes only
+        informational: true
+        paths:
+          - "bindings"
```

Comment:
The `matrix-sdk-crypto-ffi` crate is not in the `bindings` directory, which also contributes to the problem.